src/Router.js

```diff
@@ -9,7 +9,13 @@ const Router = () => {
         <Route exact path="/">
           <Home />
         </Route>
-        <Route exact path="/levels/:difficulty">
+        <Route exact path="/levels/beginner/:stage">
```

Comment:
This should be `<Route exact path"/levels/:difficulty/:stage" />`
You won't have to add a route for each different difficulty this way and you can access it in the component in an easier way.